coderd/externalauth/externalauth.go

```diff
@@ -95,17 +95,31 @@ func (c *Config) GenerateTokenExtra(token *oauth2.Token) (pqtype.NullRawMessage,
 	}, nil
 }
 
+// InvalidTokenError is a case where the "RefreshToken" failed to complete
+// as a result of invalid credentials. Error contains the reason of the failure.
+type InvalidTokenError string
```

Comment:
nit: might wanna make this a struct instead, just so we can do `e.ValidationMessage` or something!

Reply:
It's more an `InvalidationMessage`, but more specifically it could be related to refreshing, a deployment configuration (disabled refreshes), or a validation error.

In all cases, it is an `error`. And the error is related to being invalid for w/e reason. I think continuing to use it as an `err.Error()` makes sense